manifests/postgres.yaml

```diff
@@ -46,12 +46,12 @@ spec:
           initialDelaySeconds: 60
           timeoutSeconds: 5
           failureThreshold: 6
-        readinessProbe:
+        readiness-probe:
```

Comment:
I should've realised that before but I think it is better to use camel-case for being closer to kubernetes, can you change it?